source/reference/api/progress.txt

```diff
@@ -64,7 +64,25 @@ Behavior
 
 - The endpoint does not auto-refresh. To get updated status, call the
   ``progress`` endpoint again.
-  
+
+- Calls to ``/progress`` before ``mongosync`` reaches the collection
+  copy phase return 0 for ``estimatedCopiedBytes`` and 1 for 
```

Comment:
```suggestion
  copy phase returns 0 for ``estimatedCopiedBytes`` and 1 for 
```

Reply:
Here, I believe "return" is correct as "Calls" is a plural subject.
`[API] Calls return...` vs `[an API] Call returns`

Reply:
Agree, unable to resolve this comment